core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -2188,6 +2189,14 @@ public Cloud getCloud(String name) {
         return clouds.getByName(name);
     }
 
+    /**
+     * Only there to bind to /cloud/ URL.
+     */
+    @Restricted(DoNotUse.class)
+    public CloudSet getCloud() {
+        return new CloudSet();
+    }
```

Comment:
This doesn't seem necessary given `CloudSet#getUrlName`.

Reply:
I liked the natural navigation between `/cloud/` and `/cloud/foo` while retaining existing cloud URLs (even if they were not wildely used)

Reply:
As long as no cloud is called `new`. Or `index`. Or a bunch of other names causing collisions.

Reply:
No, it actually works as expected. Manually checked `/cloud/new` vs `/cloud/new/`.